modules/recent-activity/client/config/recent-activity.routes.js

```diff
@@ -144,7 +145,7 @@ angular.module('recent-activity').config(['$stateProvider', function ($stateProv
           return RecentActivityModel.getNew ();
         }
       },
-      controller: function ($scope, $state, Authentication, recentActivity, RecentActivityModel, $filter, publishedProjects) {
+      controller: function ($scope, $state, $timeout, Authentication, recentActivity, RecentActivityModel, $filter, publishedProjects) {
```

Comment:
This controller and the one below seem to be very similar, if not the same.
Could we declare the function once, and then reference it in both places (binding it to both states)?
Only if it is not a major time-consuming task.